src/components/common/SuperDownloadButton.jsx

```diff
@@ -61,12 +61,12 @@ class SuperDownloadButton extends React.Component {
     // Grommet Button determines disabled state based on presence of onClick handler...
     return (
       <Button
-        className={this.props.className}
+        className={this.props.className || null}
```

Comment:
Out of curiosity, what was happening here? When className was `undefined`, it printed the class name as the string `"undefined"` ?

Reply:
Oh, I was trying to prevent that from happening. It should be using null if this.props.className is falsey. This isn't doing what I thought it would.